common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerFactory.scala

```diff
@@ -44,7 +44,10 @@ case class ContainerPoolConfig(userMemory: ByteSize, concurrentPeekFactor: Doubl
    */
   private val totalShare = 1024.0 // This is a pre-defined value coming from docker and not our hard-coded value.
   // Grant more CPU to a container if it allocates more memory.
-  def cpuShare(reservedMemory: ByteSize) = (totalShare / (userMemory.toBytes / reservedMemory.toBytes)).toInt
+  def cpuShare(reservedMemory: ByteSize) = {
+    val cpuShare = (totalShare / (userMemory.toBytes / reservedMemory.toBytes)).toInt
```

Comment:
@KeonHee Thank you for the contribution.
How about using `max`?
Then we may be able to something like this:

```
val cpuShare = max((totalShare / (userMemory.toBytes / reservedMemory.toBytes)).toInt, 2)
```

Reply:
@style95 Thanks for your review. I think that's better!